GeneratorPgn.py

```diff
@@ -0,0 +1,318 @@
+import chess
+import torch.multiprocessing as mp
+import bz2
+from collections import deque
+import random
+import urllib
+import numpy as np
+
+class GeneratorPgn():
+    def __init__(self, replay_queue: mp.Queue, pgn_path):
+
+        """
+        Iterable class that generates stochastic fen and value samples by parallel running generators.

```

Comment:
fen -> FEN. Besides, I don't understand what is "fen and value", could you elaborate?

Reply:
I changed the description a bit to make it more relevant.